crates/core_simd/src/intrinsics.rs

```diff
@@ -0,0 +1,12 @@
+extern "platform-intrinsic" {
```

Comment:
If we have a link to LLVM's docs about how these work we should add it here.

Reply:
I found the language guide at one point--let me find that.

Reply:
Added a link to the language guide and documented which LLVM instructions are being used.